eventmesh-dashboard-common/src/main/java/org/apache/eventmesh/dashboard/common/enums/metadata/MetadataServiceTypeEnum.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.common.enums.metadata;
+
+public enum MetadataServiceTypeEnum {
```

Comment:
Better be `MetadataInstanceTypeEnum` here, since we have unified all `service_type` to `instance_type`.

Reply:
it's a mistake that these two metadata enums are not used for now.
I will remove them.